client/src/components/UI_DateTime.re

```diff
@@ -19,22 +19,12 @@ module Transformer: {
   };
 };
 
-let component = ReasonReact.statelessComponent("DateDisplay");
+let make = (~date: Js.Json.t, ~transformer=Transformer.relative, ~className=?) => {
```

Comment:
There is warning here because:

```
  22 │ let make = (~date: Js.Json.t, ~transformer=Transformer.relative, ~clas
       sName=?) => {
  23 │   let date = date |> Js.Json.decodeString;
  24 │

  This optional parameter in final position will, in practice, not be optional.
  Reorder the parameters so that at least one non-optional one is in final position or, if all parameters are optional, insert a final ().
```

Please move `date` to the last param.